src/content/docs/victory-histogram.md

```diff
@@ -0,0 +1,559 @@
+  theme={VictoryTheme.material}
+  domainPadding={{ x: 20 }}
+>
+  <VictoryHistogram
+    style={{
+      data: { fill: "#c43a31" }
+    }}
+    data={sampleHistogramData}
+    bins={[0, 3, 7, 10]}
+    // bins={5}
+  />
+</VictoryChart>
+```
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domainPadding={{ x: 20 }}
+>
+  <VictoryHistogram
+    style={{
+      data: { fill: "#c43a31" }
+    }}
+    data={sampleHistogramDateData}
+    bins={[
+      new Date(2020, 1, 1),
+      new Date(2020, 4, 1),
+      new Date(2020, 8, 1),
+      new Date(2020, 11, 1)
+    ]}
+  />
+</VictoryChart>
+```
+
+```playground_norender
+const App = () => {
+  const niceTimeScale = d3Scale
+    .scaleTime()
+    .domain(d3Array.extent(sampleHistogramDateData, ({ x }) => x))
+    .nice();
+
+  // get thresholds to bin data by months
+  const bins = niceTimeScale.ticks(d3Time.utcMonth); // try utcDay
+
+  return (
+    <VictoryChart
+      theme={VictoryTheme.material}
+      domainPadding={{ x: 20 }}
+    >
+      <VictoryHistogram
+        style={{
+          data: { fill: "#c43a31" }
+        }}
+        data={sampleHistogramDateData}
+        bins={bins}
+      />
+    </VictoryChart>
+  );
+}
+
+ReactDOM.render(<App/>, mountNode);
+```
+
+## binSpacing
+
+`type: number`
+
+The `binSpacing` prop is used to specify space between each bin. `binSpacing` represents the number of pixels that will be between each bin (including at the beginning and end of the bins).
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domainPadding={{ x: 20 }}
+>
+  <VictoryHistogram
+    binSpacing={20}
+    style={{
+      data: { fill: "#c43a31" }
+    }}
+    data={sampleHistogramData}
+  />
+</VictoryChart>
+```
+
+## containerComponent
+
+`type: element`
+
+`VictoryHistogram` uses the standard `containerComponent` prop. [Read about it here](/docs/common-props#containercomponent)
+
+```jsx
+containerComponent={<VictoryVoronoiContainer/>}
+```
+
+## cornerRadius
+
+`type: function || number || { top, bottom, topLeft, topRight, bottomLeft, bottomRight }`
+
+The `cornerRadius` prop specifies a radius to apply to each bar. If this prop is given as a single number, the radius will only be applied to the _top_ of each bar. When this prop is given as a function, it will be evaluated for each bar with the props object corresponding to that bar.
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domainPadding={{ x: 15 }}
+>
+  <VictoryHistogram
+    cornerRadius={{ topLeft: ({ datum }) => datum.y * 4 }}
+    style={{
+      data: {
+        fill: "#c43a31",
+        width: 25
+      }
+    }}
+    data={sampleHistogramData}
+  />
+</VictoryChart>
+```
+
+## data
+
+`type: array[object]`
+
+`VictoryHistogram` uses the standard `data` prop, except for it only expects each object within the array to have `x` properties. You can use the `x` data accessor prop to defined a custom data format. [Read about it here](/docs/common-props#data)
```

Comment:
Add a note about preprocessed data?

Reply:
You should also call out the limitation around qualitative data, and point users to `VictoryBar` for that use case